src/screens/WeatherScreen.js

```diff
@@ -131,6 +131,7 @@ export const WeatherScreen = () => {
               horizontal
               keyExtractor={hourlyKeyExtractor}
               renderItem={renderHourlyWeather}
+              showsHorizontalScrollIndicator={false}
```

Comment:
can you explain more in detail, what the difference is between having it `false` here compared to before? how does the look be different and how does it effect voice over?

Reply:
If we set this value to true, an indicator will appear under the FlatList. When a person using VoiceOver clicks on this area, he clicks on this indicator instead of on the degree and cannot listen to the data he wants. That's why I set the value of the indicator to false.

As you can see in the screenshot below, there is no big difference in appearance.


|true|false|
|--|--|
![IMG_0110](https://user-images.githubusercontent.com/11755668/212102779-0d78ef8e-443a-4865-b538-0d1ec4dc8e2d.jpg) | ![IMG_0111](https://user-images.githubusercontent.com/11755668/212102808-c10b47c1-643e-4997-9172-ad15510b8461.PNG)